src/components/nutrition/food/custom-food.tsx

```diff
@@ -178,6 +179,24 @@ export function CustomFoodFields() {
               />
             ))}
           </FormRow>
+          <Box marginBottom={2} />
+          <NutritionLabel
+              width={"50%"}
+              servingText={servingText}
+              calories={watch("calories")}
+              caloriesFromFat={watch("nutritionValues.caloriesFromFat")}
```

Comment:
You could pass in the entire nutritionValues object to the nutrition label as a single property so you don't have to pass each value individually